cmd/release-controller/http.go

```diff
@@ -703,7 +703,33 @@ func (c *Controller) httpReleaseInfo(w http.ResponseWriter, req *http.Request) {
 
 	renderVerifyLinks(w, *info.Tag, info.Release)
 
-	if upgradesTo := c.graph.UpgradesTo(tag); len(upgradesTo) > 0 {
+	upgradesTo := c.graph.UpgradesTo(tag)
+
+	var missingUpgrades []string
+	upgradeFound := make(map[string]bool)
+	supportedUpgrades, _ := c.getSupportedUpgrades(tagPull)
+	if len(supportedUpgrades) > 0 {
+		for _, u := range upgradesTo {
+			upgradeFound[u.From] = true
+		}
+		for _, from := range supportedUpgrades {
+			if !upgradeFound[from] {
+				upgradesTo = append(upgradesTo, UpgradeHistory{
+					From:  from,
+					To:    tag,
+					Total: -1,
+				})
+				missingUpgrades = append(missingUpgrades, fmt.Sprintf(`<a class="text-monospace" href="/releasetag/%s">%s</a>`, from, from))
+			}
+		}
+	}
+	if len(missingUpgrades) > 0 {
```

Comment:
Make this visually consistent with either the rest of the upgrade info, or make it an alert / visual warning.  I.e. either `<p class="text-warning">...`  or `<ul><li>4.4.16...`.  As is there's no reason to be different.

Reply:
Based on this feedback, I have been fooling around with a few different options...
1. Original design updated with a list
![Option1](https://user-images.githubusercontent.com/43015045/88100150-27078e80-cb6a-11ea-8d94-f2737bca69f8.png)

2. Reorganized the layout some and updated the **Untested** list to be a visual component
![Option2](https://user-images.githubusercontent.com/43015045/88100425-81a0ea80-cb6a-11ea-972b-4a9b02b3c293.png)

3.  Removed the **Untested** upgrades from the list of upgrades
![Option3](https://user-images.githubusercontent.com/43015045/88100527-a39a6d00-cb6a-11ea-869c-d4dbbfdddc91.png)

4. Switched the **Untested** block to a "danger" block
![Option4](https://user-images.githubusercontent.com/43015045/88100674-dc3a4680-cb6a-11ea-8ba6-b0fa1a28be13.png)

Personally, I like option 3 or 4 depending on how important these untested upgrades are to the consumer.
@smarterclayton @wking Any preference?

Reply:
If you're going to do an alert section, you should probably have the links inline `Untested upgrades: x, y, z`.  Only leave it as a UL if it's not in an alert section.  

It's probably a warning, not a red.  Warning + inline would be fine for me.

Reply:
Here is a screenshot of those changes:
![option5](https://user-images.githubusercontent.com/43015045/88218282-d48fa600-cc2d-11ea-8441-f96839da5bee.png)